j2parse.py

```diff
@@ -35,7 +35,7 @@ def render(tpl_path, context):
 if options.file:
     with open(options.file, 'r') as stream:
         try:
-            environment = yaml.load(stream)
+            environment = yaml.load(stream, Loader=yaml.FullLoader)
```

Comment:
For those looking for this as well, here's the relevant changelog: https://pyyaml.org/wiki/PyYAML#history

Reply:
As also seen here: https://github.com/PX4/Firmware/pull/11910

Reply:
thanks.